File: Modul 6/PRAK605/PRAK605-2310817310011-MuhammadRizky.c
#include <stdio.h>

int main() {
    int ordo;
    scanf("%d", &ordo);

    int matrixA[10][10], matrixB[10][10], result[10][10];

    printf("Matriks A:\n");
    for (int i = 0; i < ordo; i++){
        for (int j = 0; j < ordo; j++){
            scanf("%d", &matrixA[i][j]);
        }
    }

    printf("Matriks B:\n");
    for (int i = 0; i < ordo; i++){
        for (int j = 0; j < ordo; j++){
            scanf("%d", &matrixB[i][j]);
        }
    }

    for (int i = 0; i < ordo; i++){
        for (int j = 0; j < ordo; j++){
            result[i][j] = 0;
            for (int k = 0; k < ordo; k++){
                result[i][j] += matrixA[i][k] * matrixB[k][j];
            }
        }
    }

    printf("Matriks AXB:\n");
    for (int i = 0; i < ordo; i++){
        for (int j = 0; j < ordo; j++){
            printf("%d ", result[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul 6/PRAK605/PRAK605-2310817310011-MuhammadRizky.py
ordo = int(input(""))
matrixA = []; matrixB = []; result = []

print("Matriks A : ")
for i in range(ordo):
    k = list(map(int, input().split()))
    matrixA.append(k)

print("Matriks B : ")
for i in range(ordo):
    k = list(map(int, input().split()))
    matrixB.append(k)

for i in range(ordo):
    result.append([])
    for j in range(ordo):
        n = 0
        for k in range(ordo):
            n += matrixA[i][k] * matrixB[k][j]
        result[i].append(n)

print("Matriks AXB:")
for x in result:
    for k in x:
        print(k, end=" ")
    print()
